src/docs/product/sentry-basics/integrate-frontend/upload-source-maps.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Enable Readable Stack Traces in Your Errors
+title: Upload Source Maps
```

Comment:
I know this is the title that seemed to make sense here, but the first two paragraphs only talk about releases, so I'm wondering if we should say something about releases in the title? Or maybe we need to add something in those paragraphs to say that uploading source files is a necessity for releases (I think)?

Reply:
lets keep the original title